FirebaseFunctions/Sources/Callable+Codable.swift

```diff
@@ -15,7 +15,12 @@
 import FirebaseSharedSwift
 import Foundation
 
-/// A `Callable` is reference to a particular Callable HTTPS trigger in Cloud Functions.
+/// A `Callable` is reference to a particular Callable HTTPS trigger in Cloud
+/// Functions.
+///
+/// - Note: If the Callable HTTPS trigger accepts no parameters, ``Never`` can
```

Comment:
Good to know about `Never`. I knew it was around forever but didn't realize `Codable` conformance was very [recent](https://github.com/swiftlang/swift-evolution/blob/main/proposals/0215-conform-never-to-hashable-and-equatable.md).